serviceregistry/src/main/java/eu/arrowhead/core/serviceregistry/database/service/ServiceRegistryDBService.java

```diff
@@ -1020,7 +1020,7 @@ public SystemResponseDTO getSystemByNameAndAddressAndPortResponse(final String s
 		try {
 			final Optional<System> systemOptional = systemRepository.findBySystemNameAndAddressAndPort(validatedSystemName, validatedAddress, validatedPort);
 			if (systemOptional.isEmpty()) {
-				throw new InvalidParameterException("No system with name: " + systemName + ", validatedSystemName: " +  validatedAddress + " and port: " + validatedPort);
+				throw new InvalidParameterException("No system with validatedSystemName: " + systemName + ", validatedAddress: " +  validatedAddress + " and port: " + validatedPort);
```

Comment:
I'd change the
- validatedSystemName => name,
- validatedAddress => address
in string, it produces a more readable error message.
Also change the systemName variable to validatedSystemName variable.